CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.30.0] - 2022-02-12
```

Comment:
this PR seems not to be based on the latest master (29.31.0), and it includes my changes for canary which makes it hard to differentiate SLF changes. Could you close this and open a clean PR from the latest master?